src/slashcommands/ISlashCommand.ts

```diff
@@ -0,0 +1,6 @@
+// just guessing
+export interface ISlashCommand {
```

Comment:
Sorry, I changed my mind what I was typing mid typing. I meant to say the only additional thing which needs to be added is the `description` property, until https://github.com/RocketChat/Rocket.Chat/issues/7307 gets implemented and Rocketlets can take advantage of it.